openfl/utilities/fedcurv/aggregation_function.py

```diff
@@ -0,0 +1,34 @@
+"""FedCurv Aggregation function module."""
+import numpy as np
+
+from openfl.component.aggregation_functions import WeightedAverage
+from pathlib import Path
+
+
+class FedCurvWeightedAverage(WeightedAverage):
+    """Aggregation function of FedCurv algorithm.
+
+    Applies weighted average aggregation to all tensors
+    except Fisher matrices variables (u_t, v_t).
+    These variables are summed without weights.
+
+    FedCurv paper: https://arxiv.org/pdf/1910.07796.pdf
+    """
+
+    def call(self, local_tensors, db_iterator, tensor_name, fl_round, tags):
+        """Apply aggregation."""
+        if (
+            tensor_name.endswith('_u')
+            or tensor_name.endswith('_v')
+        ):
+            tensors = [local_tensor.tensor for local_tensor in local_tensors]
+            agg_result = np.sum(tensors, axis=0)
+            with open(Path('~').expanduser() / 'fedcurv_log.txt', 'a') as f:
```

Comment:
If `Path('~').expanduser()` user's home direcory maybe better to use `Path.home()`